src/gapstate.c

```diff
@@ -7,72 +7,4 @@
 
 #include "gapstate.h"
 
-#include <assert.h>
-
-typedef struct {
-    UInt size;
-    Int offset;
-    ModuleConstructor constructor;
-    ModuleDestructor destructor;
-} StateDescriptor;
-
-static StateDescriptor StateDescriptors[STATE_MAX_HANDLERS];
-static Int StateDescriptorCount = 0;
-static Int StateNextFreeOffset = 0; // Start of next free memory area (as offset into GAPState.StateSlots)
-
 GAPState MainGAPState;
```

Comment:
This is the only non-preprocessor non-comment line remaining in `gapstate.c` -- perhaps we should remove the file, and move this line somewhere else (e.g. `gap.c`?)